app/src/test/java/com/ucmobiledevelopment/freeways/IncidentUnitTest.kt

```diff
@@ -94,4 +99,30 @@ class IncidentUnitTest {
         assertTrue(containsIncidentWithCaseId400132)
     }
 
+    @Test
+    fun `Given incident API is available When I fetch incidents from 2019 to 2020 in state 40 county 1 Then results should contain a case with caseId 400434`() = runTest{
+        givenIncidentServiceIsInitialized()
+        whenIncidentDataAreReadAndParsed()
+        thenTheIncidentCollectionShouldContainIncidentWithCaseId400434()
+    }
+
+    private fun givenIncidentServiceIsInitialized() {
+        incidentService = IncidentService()
+    }
+
+    private suspend fun whenIncidentDataAreReadAndParsed() {
```

Comment:
Function names now in lowerCamelCase: https://kotlinlang.org/docs/coding-conventions.html#function-names